pages/begrippenlijsten.md

```diff
@@ -3,10 +3,14 @@ title: Begrippenlijsten
 position: 4
 ---
 
-# Begrippenlijsten gebruiken
-
 <!-- Het lijkt me een goed idee om te beginnen met een algemeen stuk over het hoe en waarom van begrippenlijsten en hun nut bij standaardisatie. Eventueel met link naar de MDTO en TOOI begrippenlijsten. -->
 
+Op deze pagina is informatie te vinden over de momenten waarop ORI-A gebruik maakt van begrippenlijsten. Een begrippenlijst is een manier om gegevens te relateren aan een extern vastgestelde (gecontroleerde) lijst van waarden. ORI-A volgt de definitie, semantiek en toepassing van begrippenlijsten zoals die ook in [MDTO](https://www.nationaalarchief.nl/archiveren/mdto/begripbegrippenlijst) worden gehanteerd.
```

Comment:
Goede definitie, denk dat we die waarschijnlijk moeten gaan hergebruiken op een andere plek.